### DISCUSSION TITLE
Use Time as X-axis in Project Insights Chart

### DISCUSSION THREAD

**Discussion by danielBingham** [community/community#54626] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

I would like to be able to construct charts with Time as the x-axis - ideally with the ability to select granularity, but using "day" as a default granluarity would be an effective MVP.  The ability to set time as the x-axis would allow me to construct many of the classic agile reports (Burndown, etc) that are currently missing from Project Insights (and is one of the things blocking my ability to convince the PM and PMO teams that Github Project is mature enough to replace Jira).  

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by slinna** [comment]
@danielBingham I just wanted to let you know that we have recently launched our [analytics tool for GitHub projects](https://screenful.com/github). It largely bridges the gap between Jira reporting and what you can get from GitHub Projects. It is now available at the [GitHub Marketplace](https://github.com/marketplace/analytics-reports). You can learn more from [this post](https://screenful.com/product-news/2023-06).
